### PR TITLE
add hertzbeat helm chart in test module

### PR DESCRIPTION
**Pull request by tomsun28**

### CODE REVIEW

File: src/test/hertzbeat/templates/manager/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+              - 'env'
+        enabled-by-default: on
+    
+    sureness:
+      auths:
+        - digest
+        - basic
+        - jwt
+      jwt:
+        secret: 'CyaFv0bwq2Eik0jdrKUtsA6bx3sDJeFV643R
```
Comment (wansir) [inline src/test/hertzbeat/templates/manager/configmap.yaml:55]: Hard code the secret key is not a good chooice.

Comment (tomsun28) [reply src/test/hertzbeat/templates/manager/configmap.yaml:55]: ok 🫡 I will fix it.

**Pull request merged**